src/rgw/CMakeLists.txt

```diff
@@ -232,6 +232,10 @@ if(WITH_JAEGER)
   add_dependencies(rgw_common jaegertracing::libjaegertracing)
 endif()
 
+if(WITH_RADOSGW_DBSTORE)
+  target_link_libraries(rgw_common PRIVATE global dbstore)
```

Comment:
@dang @kalebskeithley .. with commit#7b61667 , rgw_common is no more linked with rgw_a library and hence dbstore (rgw_sal_dbstore) had to be be linked directly to rgw_common now. 

Please let me know if dbstore should be linked to rgw_a target as well.